atoma-state/src/handlers.rs

```diff
@@ -31,11 +32,9 @@ pub async fn handle_atoma_event(
             handle_node_task_unsubscription_event(state_manager, event).await
         }
         AtomaEvent::StackCreatedEvent(event) => {
-            if event.owner == state_manager.state.owner {
-                handle_stack_created_event(state_manager, event).await
-            } else {
-                Ok(())
-            }
+            // Don't handle creation here. It's handled when the stack is created right away.
```

Comment:
Makes sense, please change it to:

`NOTE: Don't handle creation here. It's handled when the stack is created right away.`

Reply:
done